src/components/Card/index.jsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react';
+import styles from './card.module.css';
+import PropTypes from 'prop-types';
+import beautifyDate from '@/utils/beautifyDate';
+
+const Card = ({ link }) => {
+  const [beautifiedDate, beautifiedTimeDiff] = beautifyDate(link.createdAt);
+
+  const handleNavigateCardLink = useCallback(
+    (e) => {
+      if (e.target.closest('.cardAsterisk')) return;
+      window.open(link.url);
+    },
+    [link.url],
+  );
+
+  return (
+    <div className={`${styles.card}`} onClick={handleNavigateCardLink}>
+      <div className={`${styles.cardAsterisk}`}>
+        <img className={`cardAsterisk`} src={'/assets/card-asterisk.svg'} alt="Card Asterisk" />
+      </div>
+      <div className={`${styles.cardImgTop}`}>
+        <img src={link.imageSource ? link.imageSource : '/assets/image-dummy.png'} alt={link.title} />
+      </div>
+      <div className={`${styles.cardCaption}`}>
```

Comment:
```suggestion
      <div className={styles.cardCaption}>
```
이 경우엔 template literal을 사용할 필요가 없어 보여요.